public/modules/core/views/invoice.client.view.html

```diff
@@ -48,16 +49,18 @@
                                     <span ng-if=" list.invoice.status == 'not_paid' ">
                                         Not Paid
                                     </span>
-                                </td>
-                                <td style="vertical-align:middle;">
-                                    <a class="download" ng-click="retrievePDF(list.invoice.id)" id="download_{{list.invoice.id}}">
-                                        <i class="fa fa-cloud-download text-info" ng-hide="loading_{{list.invoice.id}}"></i>
-                                        <i class="fa fa-spinner fa-pulse" ng-show="loading_{{list.invoice.id}}"></i>
-                                    </a>
-                                </td>
-                            </tr>
-                        </tbody>
+                        </td>
+                        <td style="vertical-align:middle;">
+                          <a class="download" ng-click="retrievePDF(list.invoice.id)" id="download_{{list.invoice.id}}">
```

Comment:
Does download_id function follow John Pappa's naming function standards? I just ask, because I found the underscore, weird in a function.